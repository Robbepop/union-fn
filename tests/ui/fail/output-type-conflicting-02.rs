fn main() {}

#[union_fn::union_fn]
trait Foo {
    type Output = i32;
    type Output = i32;

    fn foo() -> Self::Output {}
}
