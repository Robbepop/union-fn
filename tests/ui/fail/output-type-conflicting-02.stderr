error: encountered conflicting Output types in #[union_fn] trait
 --> tests/ui/fail/output-type-conflicting-02.rs:6:5
  |
6 |     type Output = i32;
  |     ^^^^^^^^^^^^^^^^^^

error: previous Output definition here
 --> tests/ui/fail/output-type-conflicting-02.rs:5:5
  |
5 |     type Output = i32;
  |     ^^^^^^^^^^^^^^^^^^
